src/vm/assemblyspec.cpp

```diff
@@ -1464,7 +1464,7 @@ DomainAssembly *AssemblySpec::LoadDomainAssembly(FileLoadLevel targetLevel,
     }
 #endif // FEATURE_REFLECTION_ONLY_LOAD
 
-    PEAssemblyHolder pFile(pDomain->BindAssemblySpec(this, fThrowOnFileNotFound, fRaisePrebindEvents, pCallerStackMark, pLoadSecurity));
+    PEAssemblyHolder pFile(pDomain->BindAssemblySpec(this, fThrowOnFileNotFound, fRaisePrebindEvents, pCallerStackMark, pLoadSecurity, TRUE, fThrowOnFileNotFound));
```

Comment:
fUseExplicitPath is TRUE iff we need to throw an exception for incorrect path, which requires to specify the explicit path.